$NetBSD: patch-CMakeLists.txt,v 1.10 2025/05/09 22:18:38 tnn Exp $

Don't generate the "version" file. It can get picked up as a
C++ header in the current directory.

--- CMakeLists.txt.orig	2021-11-22 13:50:34.000000000 +0000
+++ CMakeLists.txt
@@ -412,8 +412,6 @@ macro(generate_versionfile)
     endif()
 endmacro(generate_versionfile)
 
-generate_versionfile()
-
 if (${SYNERGY_BUILD_LEGACY_INSTALLER})
 #
 # macOS app Bundle
